src/Components/UserComponent/UserProfile.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import { useParams } from "react-router-dom";
+import { connect } from "react-redux";
+import dateFormat from "dateformat";
+import { loadUserProfileDetail, favoritedArticleClicked } from "../../ReduxStore/FeedDetails/feedActions";
+
+const InternalUserProfile = props => {
+  const { userName } = useParams();
+
+  useEffect(() => {
+    props.loadUserProfileDetail(userName);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  
+  return (
+    <div className='profile-page'>
+      
+      {/* 用户信息 */}
+      <div className='user-info'>
+        <div className='container'>
+          <div className='row'>
+            <div className='col-xs-12 col-md-10 offset-md-1'>
+              <img
+                src={
+                  props.currentProfileData.profile &&
+                  props.currentProfileData.profile.image
+                }
+                className='user-img'
+                alt='au'
+              />
+              <h4>{userName}</h4>
+              <p>
+                {props.currentProfileData.profile &&
+                  props.currentProfileData.profile.bio}
+              </p>
+              <button className='btn btn-sm btn-outline-secondary action-btn'>
+                <i className='ion-plus-round'></i>
+                &nbsp; + Follow {userName}
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className='container'>
+        <div className='row'>
+          <div className='col-xs-12 col-md-10 offset-md-1'>
+            {/* 副导航 */}
+            <div className='articles-toggle'>
+              <ul className='nav nav-pills outline-active'>
+                <li className='nav-item'>
+                  <a className='nav-link active' href='#top'>
+                    My Articles
+                  </a>
+                </li>
+
+                <li className='nav-item'>
+                  <a className='nav-link' href='#top'
+                  
+                    onClick={(userName)=>{props.onFavoritedArticleClicked(userName)}}
+                  >
+                    Favorited Articles
+                  </a>
+                </li>
+
+
+              </ul>
+            </div>
+
+            {/* 一篇文章 */}
+            {props.currentUsersArticles.map((article,index) => {
```

Comment:
`favoritedArticles` is connected but not used here.

Reply:
fixed